packages/wallet/backend/src/grant/service.ts

```diff
@@ -80,6 +80,41 @@ export class GrantService implements IGrantService {
       args.filter = { identifier: { in: identifiers } }
     }
 
-    return await this.rafikiAuthService.listGrantsWithPagination(args)
+    const grants = await this.rafikiAuthService.listGrantsWithPagination(args)
+    this.parseIntervals(grants.grants.edges.map((it) => it.node))
```

Comment:
this type of manipulation is confusing.
I propose to change it to something like:
```
grants.grants.edges.forEach((edge) =>  {
  edge.access = this.parseAccessIntervals(edge.access)
})

.....

parseAccessIntervals(accesses) {
  return accesses.map((access) => {
    if ((access.limits?.interval) {
      return {
        ...access,
        limits: {
          ...access.limits,
          intervalHR: transformIntervaltoHR(access.limits.interval)
        }
      }
    }
    return access
  })
}
```